UpdateSamplesOutput.bat

```diff
@@ -1,5 +1,8 @@
 @echo off
 
+:: Clear previous run status
+COLOR 0F
```

Comment:
That doesn't really reset, it just sets the colors to black and white.

I think to reset you just need to call `COLOR` without any argument.

Reply:
I fixed the bat files to omit the argument, however the python script wouldn't work without the explicit 0F for some reason so I left it as is.